libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/defaults.parm

```diff
@@ -0,0 +1,8 @@
+BATT_VOLT_PIN 12
+BATT_CURR_PIN 13
+
+CAN_P1_DRIVER 1
```

Comment:
These should be set automatically I believe

Reply:
are we sure????

Reply:
I suggest remain it, if customer uses their own analog PMU, they just need to configure BATT_MONITOR = 4,  no need to further configure BATT VOLT PIN=12 and BATT CURR PIN=13, I think this would be more user-friendly.

Reply:
I think he was referring to the CAN line, and I am not sure it gets set just because a CAN port is configured....its used in many other autopilots with CAN in their defaults.param files